declarations/LoaderContext.d.ts

```diff
@@ -0,0 +1,185 @@
+import type { RawSourceMap } from "source-map";
+import type { Schema } from "schema-utils/declarations/validate";
```

Comment:
Let's avoid using `"schema-utils/declarations/validate"` in favor https://github.com/webpack/schema-utils/blob/master/declarations/index.d.ts#L3 (we export `validate`, they can be moved in theory in other place)

Reply:
So you'd rather we go with this? `import type { Schema } from "schema-utils/declarations/ValidationError";`

Reply:
Done!